### PR TITLE
TW-118: header icon android fix

### PR DESCRIPTION
**Pull request by letier3110**

### CODE REVIEW

File: src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx [hunk 1]
```diff
@@ -52,12 +53,12 @@ export const RemoveLiquidityModal = () => {
...
   const styles = useRemoveLiquidityModalStyles();
-  const lpTokenMetadata = useTokenMetadataSelector(LIQUIDITY_BAKING_LP_SLUG);
 
-  const lpToken: TokenInterface = {
-    ...emptyToken,
-    ...lpTokenMetadata
-  };
+  const token = useMemo(
+    () =>
+      tokensList.find(candidateToken => getTokenSlug(candidateToken) === LIQUIDITY_BAKING_LP_SLUG) ?? { ...emptyToken },
```
Comment (unixvb) [inline src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx:59]: why do you make copy of `emptyToken`?

File: src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx [hunk 2]
```diff
@@ -52,12 +53,12 @@ export const RemoveLiquidityModal = () => {
 
   const { publicKeyHash } = useSelectedAccountSelector();
   const styles = useRemoveLiquidityModalStyles();
-  const lpTokenMetadata = useTokenMetadataSelector(LIQUIDITY_BAKING_LP_SLUG);
 
-  const lpToken: TokenInterface = {
-    ...emptyToken,
-    ...lpTokenMetadata
-  };
+  const token = useMemo(
```
Comment (unixvb) [inline src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx:57]: rename to `lpToken`, as it was

File: src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx (hunk 1 above)
Comment (letier3110) [reply src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx:59]: because there can be no lp token

**Review by LendiHop — APPROVED** [review]

File: src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx (hunk 1 above)
Comment (unixvb) [reply src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx]: return as it was, but without destructuring `emptyToken`

**Review by unixvb — APPROVED** [review]

**Pull request merged**